### ISSUE TITLE
06-cascade-fix: update exercise to not rely on color

### ISSUE THREAD

**Issue by thatblindgeye** [TheOdinProject/css-exercises#188]
Complete the following REQUIRED checkboxes:
-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this issue follows the `location for request: brief description of request` format, e.g. `Foundations exercises: Add exercise for XYZ`

The following checkbox is OPTIONAL:

-   [ ] I would like to be assigned this issue to work on it

<hr>

**1. Description of the Feature Request:**

The 06 exercise in the foundations directory needs to be updated to not rely on color for the two groups of texts (currently green and red). One suggestion on Discord was to use black and gray or rely on font weight (200 and 900). Another option could be to have all text be black, but have one group have a background color.

The goal should be to use CSS properties mentioned in the [CSS Foundations lesson](https://www.theodinproject.com/lessons/foundations-css-foundations)

**2. Acceptance Criteria:**

- [ ] The exercise is updated to not rely on text color alone to differentiate the two groups of texts, or use a better combination of colors
- [ ] The solution image in the README is updated to reflect the new solution criteria

**3. Additional Information:**

**Comment by dm-murphy** [comment]
@Amandasilvbr Any interest in taking on this accessibility issue? If so, just let us know! 🙂

**Comment by Amandasilvbr** [comment]
Hi, @dm-murphy ! Sure, yes i'm interested!

**Comment by dm-murphy** [comment]
> Hi, @dm-murphy ! Sure, yes i'm interested!

Great! Thanks @Amandasilvbr! You have just been assigned to the issue 🎉 

Be sure to refer to the description of the request and the acceptance criteria above as you make your pull request. Please let us know if you have any questions! 🙂
